data/models/index.js

```diff
@@ -27,7 +27,7 @@ Object.keys(db).forEach(function(modelName) {
   }
 });
 
-sequelize.sync();
+sequelize.sync({force: true});
```

Comment:
Yes, this will drop all tables for an existing deployment. Yes, I plan to change this to a Sequelize data migration rather than just forcing the model sync on each startup.